src/js/menu/acquia_lift.personalize.collections.js

```diff
@@ -76,7 +76,7 @@
     },
 
     /**
```

Comment:
I *think* that the variations and the resetVariations functions are now obsolete as well.  They were convenience functions for dealing with the page variations structure.